### PR TITLE
Add run_virtual_iridium.sh

### PR DESCRIPTION
**Pull request by hhenry01**
### Description

- Add script that automates the running of the virtual iridium server https://github.com/UBCSailbot/sailbot_workspace/pull/112

### Verification

- See https://github.com/UBCSailbot/network_systems/tree/virtual-iridium-setup/scripts#run-virtual-iridium

### CODE REVIEW

File: scripts/run_virtual_iridium.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+SERVER_URL=${1:-127.0.0.1:8000}
```
Comment (patrick-5546) [inline scripts/run_virtual_iridium.sh:3]: Can we use a different port? I'm planning to add the docs site to sailbot_workspace, which uses port 8000

Comment (patrick-5546) [reply scripts/run_virtual_iridium.sh:3]: It actually opened port 8080 for me. Web hook server endpoint is 8000, http server port is 8080

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+1. `./run_virtual_iridium.sh`
+2. To monitor just the `$LOCAL_TRANSCEIVER_TEST_PORT` without extra debug messages, in a new terminal run
+    `cat $LOCAL_TRANSCEIVER_TEST_PORT`. What you see output from this command will be what the Local Transceiver reads
+    and sends.
+3. `stty 19200 < $LOCAL_TRANSCEIVER_TEST_PORT` to set the baud rate.
+4. `printf "at+sbdix\r" > $LOCAL_TRANSCEIVER_TEST_PORT`. This command queries the (currently empty) mailbox.
+5. `curl -X POST -F "test=1234" http://localhost:8080` (this is garbage data - it doesn't mean
+    anything). You should see the original terminal print that it received a POST request.
+6. `echo at+sbdix` to view the mailbox again. It will now indicate that it has the data.
```
Comment (patrick-5546) [inline scripts/README.md:36]: I don't think this command works, I just used the command in step 4 again

Comment (patrick-5546) [reply scripts/README.md:36]: Also to clarify things, you could also specify that you run the commands from step 3 onwards in a third terminal

Comment (patrick-5546) [reply scripts/README.md:36]: It worked as expected on my machine

File: scripts/run_virtual_iridium.sh (hunk 1 above)
Comment (hhenry01) [reply scripts/run_virtual_iridium.sh]: Changed to 8081. Also clarified what the variables/arguments refer to.

**Review by patrick-5546 — APPROVED** [review]

**Pull request merged**